Assets/Editor/CreatureLinker.cs

```diff
@@ -144,7 +145,7 @@ private static void LinkUnit(string unitDir)
             var effectsAnimatorField = effectsManagerScript.GetType().GetField("effectsAnimator");
             if (effectsAnimatorField != null)
             {
-                // TODO: Make this not shitty and fragile
```

Comment:
but did you? haha

Reply:
Yeah I tested this and it works, I think I had copy pasta'd the healthbar comment and this one always worked lolol